cmd/vm_start.go

```diff
@@ -8,10 +8,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// startCmd represents the start command
 var vmStartCmd = &cobra.Command{
-	Use:               "start <vm name | id>+",
-	Short:             "Start virtual machine",
+	Use:               "start NAME|ID",
+	Short:             "Start a stopped Compute instance",
```

Comment:
Not sure we should assume the Compute instance state in the command; i.e. maybe we can issue the `start` command to an already started instance, which results as a no-op command...
```suggestion
	Short:             "Start a Compute instance",
```

Reply:
@zyegfryed I've specified *stopped* instance to avoid ambiguity with the `create` command, in case some people would interpret "start" as "start a new instance". If you feel it's overkill indeed we can remove *stopped*.

Reply:
OK, got it. No problem on my side to let it as is.